Mathlib/Topology/Bases.lean

```diff
@@ -757,6 +757,23 @@ instance (priority := 100) SecondCountableTopology.to_firstCountableTopology
       ⟨(isBasis_countableBasis α).nhds_hasBasis,
         (countable_countableBasis α).mono inter_subset_left⟩⟩
 
+-- see Note [lower instance priority]
+instance (priority := 100) [Countable α] [FirstCountableTopology α] :
+    SecondCountableTopology α where
+  is_open_generated_countable := by
```

Comment:
Optional: would you like to add a one-sentence summary, like "merging the countable neighbourhood bases at each point yields a countable basis"?